<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des livres</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion_Articles</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>701</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Gestion Articles</string>
       </attribute>
       <widget class="QTableView" name="tabchambre">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>541</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: #aa0000;
    color: #fffff8;
}
QTableView{border: 6px solid #aa0000; /*other code about padding background stc*/ ;}
QHeaderView::section {
    background-color: #aaaa7f;
    padding: 4px;
    border: 1px solid #fffff8;
    font-size: 8pt;
}

QTableWidget {
    gridline-color: #ff4842;
    font-size: 9pt;
}

QTableWidget QTableCornerButton::section {
    background-color: #ff4842;
    border: 1px solid #fffff8;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>220</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>230</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Titre Article</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_num">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #b50000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>200</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id Article</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>300</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>260</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}
</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>180</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Article</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_disponibilite">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>190</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>340</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Titre event</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>340</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_num_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #b50000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>711</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(C:/Users/MY HP/Desktop/QT/projet2020/malek/afficher5.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>270</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>300</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Contenu</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_num_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>300</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #b50000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
        </property>
       </widget>
       <zorder>label_9</zorder>
       <zorder>lineEdit_disponibilite</zorder>
       <zorder>tabchambre</zorder>
       <zorder>pb_ajouter</zorder>
       <zorder>label_2</zorder>
       <zorder>lineEdit_num</zorder>
       <zorder>label</zorder>
       <zorder>pb_supprimer</zorder>
       <zorder>pb_modifier</zorder>
       <zorder>pb_rechercher</zorder>
       <zorder>label_4</zorder>
       <zorder>label_11</zorder>
       <zorder>comboBox</zorder>
       <zorder>lineEdit_num_3</zorder>
       <zorder>dateEdit</zorder>
       <zorder>label_8</zorder>
       <zorder>lineEdit_num_4</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Gestion_Event</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Gestion_Event</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>160</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Titre event</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #b50000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date event</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>250</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lieu Event</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>200</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_4">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>240</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>280</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tabservicech">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>521</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: #aa0000;
    color: #fffff8;
}
QTableView{border: 6px solid #aa0000; /*other code about padding background stc*/ ;}
QHeaderView::section {
    background-color: #aaaa7f;
    padding: 4px;
    border: 1px solid #fffff8;
    font-size: 8pt;
}

QTableWidget {
    gridline-color: #ff4842;
    font-size: 9pt;
}

QTableWidget QTableCornerButton::section {
    background-color: #ff4842;
    border: 1px solid #fffff8;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_dispo">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>200</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>160</y>
          <width>96</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
color: #ffffff;
border: 3px solid #f8d466;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #550000);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #aa3118, stop: 1 #552a2a);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #aa3118, stop: 1 #554040);
}</string>
        </property>
        <property name="text">
         <string>rechercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>711</width>
          <height>421</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(C:/Users/MY HP/Desktop/QT/projet2020/malek/afficher5.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>200</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>250</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #b50000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>300</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #b50000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>300</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Contenu Event</string>
        </property>
       </widget>
       <zorder>label_12</zorder>
       <zorder>lineEdit_dispo</zorder>
       <zorder>label_5</zorder>
       <zorder>lineEdit_id_3</zorder>
       <zorder>label_6</zorder>
       <zorder>label_7</zorder>
       <zorder>pb_ajouter_2</zorder>
       <zorder>pb_modifier_4</zorder>
       <zorder>pb_supprimer_2</zorder>
       <zorder>tabservicech</zorder>
       <zorder>pushButton</zorder>
       <zorder>dateEdit_2</zorder>
       <zorder>lineEdit_prix_3</zorder>
       <zorder>lineEdit_prix_4</zorder>
       <zorder>label_10</zorder>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Gestion des livres</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
